pom.xml

```diff
@@ -252,7 +252,7 @@
                 <artifactId>tomcat7-maven-plugin</artifactId>
                 <version>2.2</version>
                 <configuration>
-                    <port>8081</port>
+                    <port>8082</port>
```

Comment:
Ok sure

On Tuesday, April 12, 2016, Srini Kandula <notifications@github.com> wrote:

> In pom.xml
> <https://github.com/srinikandula/takewebsh/pull/6#discussion_r59390902>:
>
> > @@ -252,7 +252,7 @@
> >                  <artifactId>tomcat7-maven-plugin</artifactId>
> >                  <version>2.2</version>
> >                  <configuration>
> > -                    <port>8081</port>
> > +                    <port>8082</port>
>
> This change would cause everybody else to switch their port number. This
> should not be committed. Can you please change the phone number and push
> again ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/srinikandula/takewebsh/pull/6/files/8c4bc55f606a6f46fe9d29910938c404dab68620#r59390902>
>